src/serialize.ts

```diff
@@ -19,9 +20,15 @@ export default (sortableContainer: Element, customItemSerializer: Function = (se
   }
   // get options
   let options = _data(sortableContainer, 'opts')
+
```

Comment:
This type guard is not nessesary, as the options part will be replaced with the store.

Reply:
ok deleted